docusaurus/docs/React/message-components/attachment.mdx

```diff
@@ -103,6 +106,56 @@ const CustomAttachment = (props) => {
 </Chat>;
 ```
 
+### Image and video sizing
+
+The following section details how the width and height of images and videos uploaded from files are computed.
+
+#### Maximum size
+
+You can control the maximum size of images and videos with the [`--str-chat__attachment-max-width`](../theming/component-variables.mdx) CSS variable (available only in [theme-v2](../theming/introduction.mdx)). The value of this variable must be a value that can be computed to a valid pixel value using the [`getComputedStyle`](https://developer.mozilla.org/en-US/docs/Web/API/Window/getComputedStyle) method (for example: `300px`, `10rem`, `calc(300px - var(--margin))`, but not `100%`). If you provide an invalid value, the image and video sizing can break, which can lead to scrolling issues inside the message list (for example the message list isn't scrolled to the bottom when you open a channel).
+
+If you set an invalid value to the variable, you'll see a warning on the browser's console:
+
+<img src={AttachmentSizeWarning} width='800' />
+
+#### File size optimization
+
+Based on the CSS settings provided for images and videos, the SDK will load an image (or thumbnail image in case of a video file) with a reduced file size while still providing sufficient image quality for the given dimensions. This will result in less network traffic and faster image load for users.
+
+#### Aspect ratio
+
+The following description is applicable for [theme-v2](../theming/introduction.mdx).
+
+The SDK will try to display images and videos with their original aspect ratio, however this isn't always guaranteed (in those cases a cropped image will be displayed). Three notable exceptions are:
```

Comment:
I suggest to provide the original image to have a reference. It was not clear to me, whether any of the images have the original aspect ratio.

Reply:
I adjusted this part, is it clear now?

Reply:
:+1: